Adyen.docc/DropIn.md

```diff
@@ -31,63 +56,51 @@ let applePayment = try ApplePayPayment(countryCode: "US",
                                        currencyCode: "USD",
                                        summaryItems: summaryItems)
```

Comment:
Intent is wrong 🤓

Reply:
I can't see :D

Reply:
![Screenshot 2022-07-11 at 15 01 55](https://user-images.githubusercontent.com/2648655/178270326-4181a7e8-e2b3-443f-8baf-10d861142fa0.png)

Reply:
Sorry, GitHub UI glitch